Project.toml

```diff
@@ -45,7 +45,7 @@ LinearAlgebra = "1.9.4"
 Logging = "1.9.4"
 MuladdMacro = "0.2.1"
 NLsolve = "4"
-OrdinaryDiffEq = "6.62.0"
+OrdinaryDiffEq = "4.0.0-6.62.0"
```

Comment:
it definitely shouldn't go lower

Reply:
I was confused by the task output. It failed with 6.62.0..